nemo/core/classes/common.py

```diff
@@ -274,6 +277,8 @@ def to_config_dict(self) -> DictConfig:
             config = OmegaConf.create(config)
             OmegaConf.set_struct(config, True)
 
+            config = convert_model_config(config)
+
```

Comment:
hmmmm, I somehow think this is redundant... but maybe it is good to have it just in case also when saving config...

Reply:
Redundant is better than missing some weird edge case - its a no-op anyway if it finds an already new type config